demo/nodejs-domino/custom.ts

```diff
@@ -0,0 +1,53 @@
+import domino from 'domino'
+import './another'
+import {
+  trustedTypes,
+  TrustedTypeConfig,
+  TrustedTypesEnforcer,
+} from 'trusted-types'
+
+function acceptOnlyTrustedHTML(html: TrustedHTML) {
+  if (!trustedTypes.isHTML(html)) console.log('untrusted html', html)
+  else console.log('trusted html', html)
+}
+
+const policy = trustedTypes.createPolicy('app', { createHTML: (s) => s })
+acceptOnlyTrustedHTML('str' as any)
+acceptOnlyTrustedHTML(policy.createHTML('safe html'))
+
+const win = domino.createWindow()
+const doc = win.document
+
+const html = doc.createElement('html')
+html.appendChild(doc.createElement('div'))
+html.appendChild(doc.createElement('div'))
+html.appendChild(doc.createElement('div'))
+
+console.log(html.innerHTML)
+
+// no fail
+html.children[0].innerHTML = 'string'
+console.log(html.innerHTML)
+
+const config = new TrustedTypeConfig(false, true, ['foo'], false)
+const enforcer = new TrustedTypesEnforcer(config, win)
```

Comment:
You could also demonstrate the default policy:

```javascript
const config = new TrustedTypeConfig(false, true, ['foo', 'default'], false)
const enforcer = new TrustedTypesEnforcer(config)
enforcer.install()

const fooPolicy = trustedTypes.createPolicy('foo', { createHTML: (s) => s })

let caught = false;
try {
  html.children[0].innerHTML = 'string' // should fail
} catch (err) {
  caught = true;
}
if (caught) {
  console.log('Caught unsafe write to innerHTML property.')
} else {
  throw new Error('Didn\'t catch unsafe write to innerHTML property!');
}

// Default policy
trustedTypes.createPolicy('default', { createHTML: (s) => s + '-default'});

html.children[0].innerHTML = 'string' // should be rewritten to 'string-default'
console.log(html.children[0].innerHTML);

```

Note how I changed the exception handler too (see other comment).

Reply:
Yep, make sense.